src/applications/caregivers/components/FormFields/FacilityList.jsx

```diff
@@ -1,130 +1,103 @@
-import React, { useState, useEffect, useCallback } from 'react';
+import React, { useState, useCallback, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import {
   VaRadio,
   VaRadioOption,
 } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
-import { fetchFacilities } from '../../actions/fetchFacilities';
-import GeneralErrorAlert from '../FormAlerts/GeneralErrorAlert';
+import { focusElement } from 'platform/utilities/ui';
+import content from '../../locales/en/content.json';
 
-export const FacilityList = ({
-  input,
-  coordinates,
-  formContext,
-  id,
-  onChange,
-  value,
-}) => {
+const FacilityList = props => {
+  const { facilities, formContext, onChange, query, value } = props;
   const { reviewMode, submitted } = formContext;
-  const [facilities, setFacilities] = useState([]);
-  const [isLoading, setLoading] = useState(false);
-  const [error, hasError] = useState(false);
-  const [dirty, setDirty] = useState(false);
 
-  const errorMessages = {
-    required: 'Please select a facility',
-  };
+  const [dirty, setDirty] = useState(false);
 
-  const handleChange = event => {
+  const handleChange = e => {
+    onChange(e.detail.value);
     setDirty(true);
-    onChange(event.detail.value);
   };
 
   const showError = () =>
-    (submitted || dirty) && !value ? errorMessages.required : false;
+    (submitted || dirty) && !value
+      ? content['validation-facialities--default-required']
+      : null;
 
   const getFacilityName = useCallback(
-    val => facilities.find(f => f.id.split('_').pop() === val)?.name || '—',
+    val => {
+      const facility = facilities.find(f => f.id.split('_').pop() === val);
+      return facility?.name || '&mdash;';
+    },
     [facilities],
   );
 
   const formatAddress = ({ address1, address2, address3 }) => {
-    // Create an array of address parts
     const parts = [address1, address2, address3];
-
-    // Filter out falsy values (null, undefined, etc.)
-    const validParts = parts.filter(part => part);
-
-    // Join the valid parts with a comma and space
+    const validParts = parts.filter(Boolean);
     return validParts.join(', ');
   };
 
+  const facilityOptions = facilities.map(f => (
+    <VaRadioOption
+      key={f.id}
+      name="facility"
+      label={f.attributes.name}
+      value={f.id.split('_').pop()}
+      description={formatAddress(f.attributes.address.physical)}
+      tile
+    />
+  ));
+
   useEffect(
     () => {
-      if (coordinates) {
```

Comment:
Moved this to the `facilitySearch` parent component to simplify this logic down.